packages/quick_actions/quick_actions_android/android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,6 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why remove this line?

Reply:
Ops, it was removed by my IDE while I commit it.